index.html

```diff
@@ -14,6 +14,138 @@
+    <br> <br>  Thirdly, while this ties into the second it bears its own mention. I have received awards in my educational and work background through a demonstration of perserverence, adaption to change, and managing my staff in a way they too see the value in implementing these qualities.</p>
+  </div>
+    </section>
+
+  <section class="ski">   <!-- Skills content opening -->
+    <h2 class="ski-h2">Skills</h2>
+    <table class="ski-table">
+      <tr class="ski1strow">
+       <th>Skill</th>
+       <th>fluency</th>
+       <th>Notables</th>
+      </tr>
+      <tr class="ski2ndrow">
+        <th>Html & CSS</th>
+        <th>Moderate fluency</th>>
+        <th>Learned during my tenure at <br> American Intercontinental University and Lambda.</th>
+      </tr>
+       <tr class="ski3rdrow">
+        <th>Js</th>
+        <th>Familiar(in current study)</th>
+        <th>Learning and applying in real world style learning enviroment</th>
+       </tr>
+       <tr class="ski4throw">
+        <th>C#, SQL, </th>
+        <th>Familiar </th>
+        <th>(learned within my bachelors at American Intercontinental University, but have not applied it in a real world setting and am a bit rusty due to lack of application and time since using</th>
+       </tr>
+    </table>
+</section><!-- Skills Closure -->
+
+  <section class="edu">   <!-- Education content opening -->
+  <h2 class="edu-h2">Education</h2>
+  <table class="edu-table">
+    <tr class="edu1strow">
+     <th>Institution</th>
+     <th>Major</th>
+     <th>Date of completion</th>
+     <th>Notables</th>
+    </tr>
+    <tr class="edu2ndrow">
+      <th>Lambda School</th>
+      <th>Full Stack Web Development</th>
+      <th>Oct 2019</th>
+      <th>Currently attending this institutions online course. This college can be equated to real world experience and educational. I maintained full time employment while attending.</th>
+    </tr>
+     <tr class="edu3rdrow">
+       <th>American Intercontinental University</th>
+       <th>Bachelors in Information Technology Programming Focus</th>
+       <th>Apr 2013</th>
+       <th>I attended this school online for the full term without breaks and maintained full time employment. Graduated with Summa Cum Laude honors</th>
+     </tr>
+     <tr class="edu4throw">
+      <th>American Intercontinental University</th>
+      <th>Associate in Business computer programming Focus</th>
+      <th>Apr 2011</th>
+      <th>I attended this school online for the full term without breaks and maintained full time employment. Graduated with Summa Cum Laude honors</th>
+     </tr>
+  </table>
+
+</section> <!-- Education closure -->
+<section class="exp">
+  <h2 class="exp-h2">Experience</h2>
+  <table class="exp-table">
+    <tr class="exp1strow">
+     <th>Employer</th>
```

Comment:
Wow, you really know your tables. Looks good. Also, I had know idea that you had such an impressive background. Very cool Thomas!